staramr/results/AMRDetectionSummary.py

```diff
@@ -115,7 +114,13 @@ def create_summary(self, include_negatives: bool=False) -> DataFrame:
         if plasmid_frame is not None:
             plasmid_frame = self._compile_plasmids(plasmid_frame)
 
-            resistance_frame = resistance_frame.merge(plasmid_frame, on='Isolate ID', how='left').fillna(value={'Plasmid Genes': 'None'})
+            if resistance_frame.empty:
```

Comment:
Could you message me an example file of when you were running into this error? I'm surprised it's happening since I would have assumed the `include_negatives()` call a few lines above would have made sure that `resistance_frame` is never empty.

Reply:
I'm mainly running into this error when running my integration test.

Reply:
Okay. That sounds good. It can be left as is.  Thanks.